satpy/scene.py

```diff
@@ -900,8 +892,7 @@ def load(self, wishlist, calibration=None, resolution=None,
             missing = self.missing_datasets.copy()
             self._remove_failed_datasets(keepables)
             missing_str = ", ".join(str(x) for x in missing)
-            LOG.warning(
-                "The following datasets were not created: {}".format(missing_str))
+            LOG.debug("The following datasets were not created: {}".format(missing_str))
```

Comment:
I think this is pretty important info, I'd like to keep this as a warning. What typically happens is that at the next step the user tries to show or save the image and it crashes. Having the reason visible above is a good thing.

Reply:
Hhhhmm how about INFO level? Warning seems negative and this isn't really a bad thing necessarily. A little iffy. Either way I can add information like "The following datasets were not created, but will be created after resampling: {}". If this was the message does that change your mind about what the log level should be?